infrastructure/src/main/java/org/corfudb/infrastructure/CorfuServer.java

```diff
@@ -477,6 +481,16 @@ protected void initChannel(@Nonnull Channel ch) throws Exception {
                     ch.pipeline().addLast("sasl/plain-text", new
                             PlainTextSaslNettyServer());
                 }
+
+                Lz4FrameEncoder encoder = new Lz4FrameEncoder(LZ4Factory.fastestInstance(),
```

Comment:
This initialization is repeated on client side as well . Can we do it in one place. Are there any common utils or does the packaging prevent it ?